src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -207,6 +205,7 @@ const UsersFragment = ({
             routeName: route.name,
             isUser
           }}
+          withKarma
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `withKarma` has been added to the `DomainList` component without any value assigned to it. This might cause issues if the `DomainList` component expects a specific type for the `withKarma` prop. If `withKarma` is a boolean prop, it would be clearer to explicitly set it to `true`.
```diff
- withKarma
+ withKarma={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->